README.md

```diff
@@ -38,21 +38,28 @@ This process will install the [Nix package manager](https://nixos.org/nix/). If
     git clone https://github.com/reflex-frp/reflex-platform
     ```
 
-2. Navigate into the `reflex-platform` folder and run the `try-reflex` command. This will install Nix, if you don't have it already, and use it to wrangle all the dependencies you'll need and drop you in an environment from which you can use Reflex. Be warned, this might take a little while the first time:
+1. Navigate into the `reflex-platform` folder and run the `try-reflex` command. This will install Nix, if you don't have it already, and use it to wrangle all the dependencies you'll need and drop you in an environment from which you can use Reflex. Be warned, this might take a little while the first time:
 
     ```bash
     cd reflex-platform
     ./try-reflex
     ```
 
-3. From this nix-shell, you can compile any haskell source files you like using ghcjs:
-
-    ```bash
-    ghcjs --make your-source-file.hs
-    ```
-    Replace `your-source-file.hs` with the name of the file you'd like to compile.  For the most part, ghcjs supports the same options as ghc.
-
-4. Compilation will produce a `your-source-file.jsexe` folder containing an `index.html` file. Open that in your browser to run your app.
+1. From this nix-shell, you can compile any haskell source files you like.
+   Replace `your-source-file.hs` with the name of the file you'd like to compile.  For the most part, ghcjs supports the same options as ghc:
+
+   * GHC
+     ```bash
+     ghcjs --make your-source-file.hs
```

Comment:
this line seems off to me. ghc --make .....?

Reply:
@alexfmpe I think @MarisaKirisame is right here

Reply:
Oops, copy-paste strikes again
Fixed